### PR TITLE
Autosuggest: Allow option for prepending empty message, defaulted to old behavior

### PR DESCRIPTION
**Pull request by jrefano**

### CODE REVIEW

File: be/autosuggest.js [hunk 1]
```diff
@@ -26,17 +26,19 @@
         return items.some(function(value) {
           return term === String(value.label || value.value || value).toLowerCase();
         });
-      }
+      },
+      prepend: false
```
Comment (mikesherov) [inline be/autosuggest.js]: would prefer a default of true, so `append` true

File: be/autosuggest.js [hunk 2]
```diff
@@ -26,17 +26,19 @@
...
     },
 
     _noMatch: function(ul, items) {
-      var templater = this.options.noMatchTemplate;
+      var templater = this.options.noMatchTemplate,
+          prepend = this.options.prepend;
```
Comment (mikesherov) [inline be/autosuggest.js]: new line after

File: be/autosuggest.js (hunk 1 above)
Comment (mikesherov) [inline be/autosuggest.js]: @jrefano wins

Comment (mikesherov) [inline be/autosuggest.js]: fatality

**Comment by mikesherov** [comment]
:+1:

File: be/autosuggest.js [hunk 3]
```diff
@@ -26,17 +26,19 @@
...
       if ($.isFunction(templater)) {
         $(templater({
           term: this.term,
           items: items
         }))
-        .appendTo(this.menu.element)
+        [prepend ? 'prependTo' : 'appendTo'](this.menu.element)
```
Comment (mrjoelkemp) [inline be/autosuggest.js]: You're doing this to keep the chain, but it would be clearer as separate statements.

File: be/autosuggest.js (hunk 1 above)
Comment (jrefano) [inline be/autosuggest.js]: http://media.giphy.com/media/433IpqycHzvkk/giphy.gif

Comment (jrefano) [inline be/autosuggest.js]: giphy

**Comment by mrjoelkemp** [comment]
:+1:

**Pull request merged**